web/locales/it.json

```diff
@@ -1748,7 +1752,7 @@
         },
         "VI": {
             "countryName": "Stati Uniti d'America",
-            "zoneName": "Isole Vergini"
+            "zoneName": "Isole Vergini statunitensi"
```

Comment:
They are known as "Isole Vergini americane"

Example from *La Farnesina*

<img width="472" alt="Screenshot 2020-10-12 at 18 22 47" src="https://user-images.githubusercontent.com/122919/95769360-0d349e00-0cb8-11eb-8b6c-844195fd0924.png">

Reply:
Another name for the Virgin Islands of the United States can be the one that I've suggested. Furthermore, "Americano" it's an adjective that have to be referred to the whole American continent, while the Virgin Islands are an U.S.A. territory. I know that in the common language U.S.A. are often referred as only "America" but it's simply wrong.

Reply:
I know, there will never be a convention in naming 😄 - anyhow, I am fine to settle on `Isole Vergini statunitensi` 😉